ExploreASL_Initialize.m

```diff
@@ -126,7 +126,7 @@
     end
     
     % Read file
-    if exist(x.DataParPath,'file')
+    if exist(x.DataParPath,'file')==2
```

Comment:
No need to add the `==2`, that is what the `'file'` does already.
I would remove the `==2` but perhaps it doesn't harm?

Reply:
I prefer the option with `==2` because ...
```matlab
exist(fullfile(server,'Server_xASL','ExploreASL','ExploreASL.m'),'file')==2
```
is `TRUE`, whereas ...
```matlab
exist(fullfile(server,'Server_xASL','ExploreASL','ExploreASL.m'),'file')
```
is `2`. And within `if-else`-statements, it kind of makes sense to my to ask...
```matlab
if (TRUE)
    then do this
else
    do this
end
```
... instead of ...
```matlab
if (2)
    then do this
else
    do this
end
```